File: message/src/main/java/com/cxyz/message/starter/IconifyStarter.java
package com.cxyz.message.starter;

import android.content.Context;

import com.cxyz.context.starter.Starter;
import com.joanzapata.iconify.Iconify;
import com.joanzapata.iconify.fonts.EntypoModule;
import com.joanzapata.iconify.fonts.FontAwesomeModule;
import com.joanzapata.iconify.fonts.IoniconsModule;
import com.joanzapata.iconify.fonts.MaterialCommunityModule;
import com.joanzapata.iconify.fonts.MaterialModule;
import com.joanzapata.iconify.fonts.MeteoconsModule;
import com.joanzapata.iconify.fonts.SimpleLineIconsModule;
import com.joanzapata.iconify.fonts.TypiconsModule;
import com.joanzapata.iconify.fonts.WeathericonsModule;

import cn.jpush.android.api.JPushInterface;
import cn.jpush.im.android.api.JMessageClient;

public class IconifyStarter implements Starter {
    @Override
    public void load(Context context) {
        //初始化sdk
        JPushInterface.setDebugMode(true);//正式版的时候设置false，关闭调试
        JMessageClient.setDebugMode(true);
        JMessageClient.init( context);
        JPushInterface.init(context);
        Iconify.with(new FontAwesomeModule())
                .with(new EntypoModule())
                .with(new TypiconsModule())
                .with(new MaterialModule())
                .with(new MaterialCommunityModule())
                .with(new MeteoconsModule())
                .with(new WeathericonsModule())
                .with(new SimpleLineIconsModule())
                .with(new IoniconsModule());
    }
}

File: main/src/main/java/com/cxyz/main/starter/IconifyStarter.kt
package com.cxyz.main.starter

import android.content.Context
import com.cxyz.context.starter.Starter
import com.joanzapata.iconify.Iconify
import com.joanzapata.iconify.fonts.*
import com.qmuiteam.qmui.widget.grouplist.QMUIGroupListView

class IconifyStarter : Starter {
    override fun load(context: Context) {
        Iconify.with(FontAwesomeModule())
            .with(EntypoModule())
            .with(TypiconsModule())
            .with(MaterialModule())
            .with(MaterialCommunityModule())
            .with(MeteoconsModule())
            .with(WeathericonsModule())
            .with(SimpleLineIconsModule())
            .with(IoniconsModule())

    }
}